### DISCUSSION TITLE
Remove author name from custom cards

### DISCUSSION THREAD

**Discussion by bavo** [UI-Lovelace-Minimalist/UI#240] [General]
Guy's, another discussion i might want to start: 

How do you feel about using the authors name in the custom_card they develop? 
I feel like it obfuscates the cards a bit, makes them difficult to find and makes it easy to create duplicates without knowing. Imo it's better to have 1 media_player card with some options instead of having 3 different cards spread out over the repo. 
Bonus: The HACS changes might be an opportunity to add breaking changes? 

What do you think?

**Comment by CM000n** [comment]
I also find the names of the creators rather confusing and unnecessary. The creators could simply immortalise themselves somewhere in the readme.
There is also still an open discussion about where we want to put the custom_cards in general. For a delivery via HACS, they are too many and too confusing if the user does not need them. The question here is whether we don't want to put them in a separate repo?
We should also go over them again to see which ones are really "custom" and which ones make sense to integrate into the standard templates.

**Comment by saxel** [comment]
I don't think it was about immortalize but rather an easy way to avoid name conflicts of cards when two authors have different takes/needs of a card for an entity of the same type. It was the naming convention as mentioned here: https://ui-lovelace-minimalist.netlify.app/development/custom_card

I think it is better if the author name is removed from directory/and yaml code.

For the location of the custom_cards I wonder what are the drawbacks of having them in the same repository?

**Reply by willyp713** [reply]
Isn't it common practice in other projects to keep authorship restricted to comment blocks within the code? If the author wants to create an alternative card, they have to come up with a unique name to avoid conflicts.

I realize this is somewhat annoying, but it is better than typing the author's name into all your code blocks. At least this way it uses real words instead of names:

- `custom_card_light`
- `custom_card_super_light`
- `custom_card_mega_light`
- `custom_card_light_mini`
- etc...

I would prefer using those ^^^ instead of having to use something like `custom_card_willyp713_light_mini`.

**Reply by saxel** [reply]
I guess your reply isn't a comment on my comment since we are on the same page. :) In my current and past projects we clear the original author if we find such a tag in the code. The git history is really sufficient!

**Comment by CM000n** [comment]
I think s wasn't about having any real disadvantages by leaving them in the same repo. It was more about making it clearer, even through a physical separation, that they are not included in the custom component by default.
But I would also be fine with leaving them in a separate folder at the top level of this repo, at least for now. At the moment we don't have the possibility to create a new repo within the org anyway, as long as paddy doesn't get back to us ;-)
